src/main/java/uk/gov/register/core/RegisterContext.java

```diff
@@ -97,12 +93,14 @@ public void transactionalRegisterOperation(Consumer<Register> consumer) {
 
     public static void useTransaction(DBI dbi, Consumer<Handle> callback) {
         try {
-            dbi.useTransaction(TransactionIsolationLevel.SERIALIZABLE, (handle, status) -> {
-                handle.getConnection().setClientInfo("ApplicationName", "openregister_transactional");
```

Comment:
a) I didn't think it was useful, no
b) I don't know whether it permanently renames the connection (so it keeps the `_transactional` name even after returning to the pool) which would do more harm than good